### PR TITLE
Dasharo ectool

### PR DESCRIPTION
**Pull request by mkopec**
For convenience, print IP addresses when enabling SSH, so we don't have to drop to shell to check manually every time

### CODE REVIEW

**Comment by mkopec** [comment]
The new IP addresses line will look something like this:

```
Your IP addresses: 192.168.4.145/24 172.17.0.1/16 192.168.4.119/24
```

**Review by miczyg1 — APPROVED** [review]

**Comment by macpijan** [comment]
@mkopec If we have to change the name of the tool right now as well, we should also change it in other places where it might have been used.
Definitely dasharo/docs, probably also dasharo/open-source-firmware, maybe someplace else as well

File: meta-dts-distro/recipes-dts/packagegroups/packagegroup-dts.bb [hunk 1]
```diff
@@ -45,7 +45,7 @@ RDEPENDS:${PN}-tools-dts = " \
     dasharo-deploy \
     dasharo-hcl-report \
     dts \
-    system76-ectool \
+    dasharo-ectool \
```
Comment (macpijan) [inline meta-dts-distro/recipes-dts/packagegroups/packagegroup-dts.bb:48]: @mkopec Does it work? I would guess the system76-ectool_0.3.8 must be renamed here as well.

**Comment by mkopec** [comment]
Yes, I planned on doing that in subsequent PRs

File: meta-dts-distro/recipes-dts/packagegroups/packagegroup-dts.bb (hunk 1 above)
Comment (mkopec) [reply meta-dts-distro/recipes-dts/packagegroups/packagegroup-dts.bb:48]: No, I'm working on fixing these issues and will submit a PR soon

**Comment by mkopec** [comment]
> @mkopec If we have to change the name of the tool right now as well, we should also change it in other places where it might have been used.

I thought about symlinking the tool to the old name so that any scripts and bits of documentation referring to the old name keep working

**Comment by macpijan** [comment]
@mkopec 

That would work as well if we include symlink in DTS. In the long run, we should still update names in documentation, as upstream tool (e.g. from fedora package) would not work.

**Pull request merged**